Create ui-c8y-1021-62-3-new-user-does-not-see-the-qr-code-when-setting-it.md (#3221)
Cumulocity-IoT/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-62-3-new-user-does-not-see-the-qr-code-when-setting-it.md

---
date: ""

> title: TOTP 2FA QR code not displayed for new users during setup

Reviewer: ```suggestion
title: TOTP QR code is properly displayed during setup
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63297
version: 1021.62.3
---

> New user does not see the QR code when setting it up with TOTP 2FA [GRAFT][release/cd] (#8899)

Reviewer 2: ```suggestion
New user (or user with TOTP secret revoked with user settings) was not able to set up TOTP because of lack of QR code. Issue was fixed, QR code is again visible and user can set up TOTP.
```

> New user (or user with TOTP secret revoked with user settings) was not able to set up TOTP because of lack of QR code. Issue was fixed, QR code is again visible and user can set up TOTP.

Reviewer: ```suggestion
New users (or users with the TOTP secret revoked in the user settings) were not able to set up TOTP because the QR code was missing. This issue has been fixed, and the QR code is again visible so that users can set up TOTP.
```
